Wikipedia/Code/PlaceSearchFilterListController.swift

```diff
@@ -50,21 +50,15 @@ class PlaceSearchFilterListController: UITableViewController {
         if (indexPath.row == 0) {
             myCell.titleLabel.text = localizedStringForKeyFallingBackOnEnglish("places-filter-top-articles")
             myCell.subtitleLabel.text = localizedStringForKeyFallingBackOnEnglish("places-filter-top-articles-count").replacingOccurrences(of: "$1", with: String(delegate.placeSearchFilterListController(self, countForFilterType: .top)))
-            
-            if (currentFilterType == .top) {
-                myCell.iconImageView?.image = #imageLiteral(resourceName: "places-suggestion-top")
-            } else {
-                myCell.iconImageView?.image = #imageLiteral(resourceName: "places-filter-top-disabled")
-            }
-
+            myCell.iconImageView?.image = #imageLiteral(resourceName: "places-suggestion-top")
         } else if (indexPath.row == 1) {
             myCell.titleLabel.text = localizedStringForKeyFallingBackOnEnglish("places-filter-saved-articles")
-            myCell.subtitleLabel.text = localizedStringForKeyFallingBackOnEnglish("places-filter-saved-articles-count").replacingOccurrences(of: "$1", with: String(delegate.placeSearchFilterListController(self, countForFilterType: .saved)))
-            
-            if (currentFilterType == .saved) {
+            let savedCount = delegate.placeSearchFilterListController(self, countForFilterType: .saved)
```

Comment:
Do the saved articles get locations added somewhere before this count is taken? Users with existing libraries will have saved articles in the db that, even though they have locations on Wikipedia, don't reflect that in core data (since we weren't fetching it previously)